airflow/providers/google/BACKPORT_README.md

```diff
@@ -0,0 +1,24 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
+
+* Kubernetes Pod operators imports are refactored to contrib.kubernetes
+* BigQuery operator is removed until it is refactored.
```

Comment:
```suggestion
* BigQuery operators are removed until it is refactored.
```

Reply:
Does this mean that when I install backport packages I will have no BQ operators?

Reply:
This is the expected result. I think it is worth adding information that we recommend using airflow.contrib operators and that new ones will be released soon.

Reply:
I agree about adding this info. Otherwise, as a user I would suspect that after installation I will have no BQ operators, including those from contrib.

Reply:
Updated. Please check :)